app/controllers/api/v2/reports_controller.rb

```diff
@@ -7,7 +7,7 @@ class ReportsController < V2::BaseController
       before_filter :find_resource, :only => %w{show update destroy}
       before_filter :setup_search_options, :only => [:index, :last]
 
-      add_smart_proxy_filters :create, :features => ReportImporter.report_features
+      add_smart_proxy_filters :create, :features => ReportImporter.authorized_smart_proxy_features
```

Comment:
Oh, regarding my comment about deprecation of authorized_smart_proxy_features, did you mean to actually keep `ReportImporter.report_features` here which calls that method on each of the known report importers?